pkgs/by-name/co/commafeed/package.nix

```diff
@@ -79,21 +83,14 @@ maven.buildMavenPackage {
     runHook preInstall
 
     mkdir -p $out/bin $out/share
-    install -Dm644 commafeed-server/target/commafeed.jar $out/share/commafeed.jar
-    install -Dm644 commafeed-server/config.yml.example $out/share/config.yml
+    unzip -d $out/share/ commafeed-server/target/commafeed-$version-h2-jvm.zip
 
     makeWrapper ${jre}/bin/java $out/bin/commafeed \
-      --add-flags "-jar $out/share/commafeed.jar"
+      --add-flags "-jar $out/share/commafeed-$version-h2/quarkus-run.jar"
```

Comment:
Do we need `--set-default QUARKUS_DATASOURCE_JDBC_URL 'jdbc:h2:./database/db;DEFRAG_ALWAYS=TRUE'` to maintain the previous db url? The [new default](https://github.com/Athou/commafeed/blob/e5c271ca1c26a1f3f73e0a0bcb0205746edb43b4/commafeed-server/src/main/resources/application.properties#L71) seems to be `./data/db` and I think if we keep the db at the same path it'll migrate it (haven't tried that though).